### DISCUSSION TITLE
Moving from Linux to Windows

### DISCUSSION THREAD

**Discussion by iamsae** [ppy/osu#22851] [Q&A] [answered]
I've recently moved from Linux (Ubuntu 20.04LTS) to Windows(Build 22621.963).

Since I was playing on osu!lazer for a long time, it had all my beatmaps/skins/scores in it.
 Although, before moving I copied the osu folder from `/home/.local/share/osu` but I'm not sure where to put this folder. Any Ideas?

**Comment by peppy** [comment, marked as answer]
You can either move it to the standard location (`%appdata%\osu`) or put it anywhere you want, then use the in-game data directory selector to point the game to it (it will then give you the option to use the existing data):

![osu! 2023-03-13 at 06 32 31](https://user-images.githubusercontent.com/191335/224624786-8e759e45-94fc-4027-8f01-04e8cdb62626.png)

**Reply by iamsae** [reply]
Worked like a charm! Thanks Peppy <3
